config/routes.rb

```diff
@@ -36,4 +38,13 @@
   get "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinary_offices#veterinarians"
   get "/veterinary_offices/:veterinary_office_id/veterinarians/new", to: "veterinarians#new"
   post "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinarians#create"
+
+  post "/applications", to: "applications#create"
+  get "/applications/new", to: "applications#new"
+  get "/applications/:id", to: "applications#show"
+  post "/applications/:id", to: "pet_applications#create"
+  patch "/applications/:id", to: "applications#update"
+
+  get "/admin/applications/:id", to: "admin_applications#show"
+  patch "admin/applications/:id", to: "pet_applications#update"
```

Comment:
You would want the path for this to be something like: 
```ruby
patch "/applications/:application_id/pet/:pet_id"
or
patch "/application_pets"
or 
patch "/applications/:id/pets" and send the pet_id as a param
```
This will give us the ids that we need to show the relationship between the two.